src/bionics_ui.cpp

```diff
@@ -145,8 +145,11 @@ static std::string build_bionic_poweronly_string( const bionic &bio )
     if( !bio.has_flag( "SAFE_FUEL_OFF" ) && !bio.info().fuel_opts.empty() ) {
         properties.push_back( _( "(fuel saving ON)" ) );
     }
-    if( bio.has_flag( "AUTO_START_ON" ) && !bio.info().fuel_opts.empty() ) {
-        properties.push_back( _( "(auto start ON)" ) );
+    if( bio.is_auto_start_on() && !bio.info().fuel_opts.empty() ) {
+        const std::string label = _( "(auto start < " ) + std::to_string( static_cast<int>
```

Comment:
I wonder why you don't use `string_format` for this. Something like `string_format( _( "(auto start < %s %%)" ), bio.get_auto_start_thresh() * 100 )`. I believe you don't even need to cast to int for this to work.